src/main/java/ro/fortsoft/pf4j/update/DefaultUpdateRepository.java

```diff
@@ -51,7 +51,7 @@ public String getId() {
     }
 
     @Override
-    public String getLocation() {
+    public String getUrl() {
```

Comment:
Yes. 
What about the constructor? Are there benefits in keeping the `String url` arg?